content/go/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: 'Conditionals'
+Description: 'Conditionals are used to evaluate if an expression is `true` or `false` and execute code based on that value.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Comparison'
+  - 'Booleans'
+  - 'If'
+  - 'Else'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+Conditionals are used to evaluate if an expression is `true` or `false` and execute code based on that value. There are several statements that use conditional expressions to determine what code to execute. The primary one being the `if...else if...else` statements.
+
+Other statements that can use conditional statements are [`switch`](https://www.codecademy.com/resources/docs/go/switch) and [`for`](https://www.codecademy.com/resources/docs/go/loops).
+
+## The `if` Statement
+
+One of the statements that's thought of most often in connection with conditional statements is the `if` statement. In its most basic form, the `if` statement evaluates if a condition is `true`, and if so, executes its associated code block.
+
+### Syntax
+
+If `condition` is `true`, execute `statements`. Execution continues after the `if` block regardless of the value of `condition`.
+
```

Comment:
Maybe it could include info about [using `if`s with short statements](https://go.dev/tour/flowcontrol/6)?

```pseudo
if small_statement; condition {
  statements
}
```